source/guides/references/error-messages.md

```diff
@@ -535,6 +534,29 @@ This error likely appeared because:
 1. You do not have internet. Please ensure you have connectivity then try again.
 2. You are a developer that has forked our codebase and do not have access to run our API locally. Please read more about this in our {% url "contributing doc" https://on.cypress.io/contributing %}.
 
+## <a id="bad-policy-settings"></a> {% fa fa-exclamation-triangle red %} Cypress detected policy settings on your computer that may cause issues
+
+When Cypress launches Chrome, it attempts to launch it with a custom proxy server and browser extension. Certain group policies (GPOs) on Windows can prevent this from working as intended, which can cause tests to break.
+
+If your administrator has set any of the following Chrome GPOs, it can prevent your tests from running in Chrome:
+
+- Proxy policies: `ProxySettings, ProxyMode, ProxyServerMode, ProxyServer, ProxyPacUrl, ProxyBypassList`
+- Extension policies: `ExtensionInstallBlacklist, ExtensionInstallWhitelist, ExtensionInstallForcelist, ExtensionInstallSources, ExtensionAllowedTypes, ExtensionAllowInsecureUpdates, ExtensionSettings, UninstallBlacklistedExtensions`
+
+Here are some potential workarounds:
+
+1. Ask your administrator to disable these policies so that you can use Cypress with Chrome.
+2. Use the built-in Electron browser for tests, since it is not affected by these policies. {% url 'See the guide to launching browsers for more information.' launching-browsers#Electron-Browser %}
+3. Try using Chromium instead of Google Chrome for your tests, since it may be unaffected by GPO. You can [download the latest Chromium build here.](https://download-chromium.appspot.com/)
```

Comment:
`[download the latest Chromium build here.](https://download-chromium.appspot.com/)` -> `{% url "download the latest Chromium build here." https://download-chromium.appspot.com/ %}`